src/kernel/pmm.zig

```diff
@@ -20,7 +21,7 @@ const PmmError = error{
 /// The size of memory associated with each bitmap entry
 pub const BLOCK_SIZE: usize = arch.MEMORY_BLOCK_SIZE;
 
-var bitmap: PmmBitmap = undefined;
+var the_bitmap: PmmBitmap = undefined;
```

Comment:
Perhaps we could start to move to the newer way of globals where this will just be: `bitmap: PmmBitmap,`. As when importing `pmm`, it is already a struct.
`std.mem.Allocator` is an example.

Reply:
I haven't heard of this new way of using globals, could you share alink?

Reply:
https://github.com/ziglang/zig/blob/master/lib/std/mem/Allocator.zig

Reply:
Perhaps this can be made into an issue as this doesn't relate to this PR and can be done as a separate PR

Reply:
Oh that's pretty cool! I agree with it being an issue instead.